mne_neuron/dipole.py

```diff
@@ -245,3 +244,19 @@ def baseline_renormalize(self, params):
         # recalculate the aggregate dipole based on the baseline
         # normalized ones
         self.dpl['agg'] = self.dpl['L2'] + self.dpl['L5']
+
+    def write(self, fname = 'dpl.txt'):
```

Comment:
I wouldn't give it a default filename actually. You will start having files called dpl.txt everywhere which doesn't seem very clean ...

Reply:
Sure, agreed.